src/api/v4/filter/bounding-box-leaflet.js

```diff
@@ -53,4 +57,9 @@ BoundingBoxLeaflet.prototype.$getInternalModel = function () {
   return this._internalModel;
 };
 
+// Helper to check if an element is a leafletmap object
+function _isLeafletMap (element) {
+  return element instanceof L.map;
```

Comment:
Be careful! You need to use `L.Map` instead of L.map (that is the function)

Reply:
Same as with `google` above.